test/src/org/labkey/test/tests/targetedms/TargetedMSChromatogramOptimizationTest.java

```diff
@@ -0,0 +1,133 @@
+package org.labkey.test.tests.targetedms;
+
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.categories.MS2;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.targetedms.ConnectionSource;
+
+import java.io.File;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Category({DailyB.class, MS2.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 6)
+public class TargetedMSChromatogramOptimizationTest extends TargetedMSTest
+{
+    private static final String SKY_FILE = "SmallMoleculeLibrary3.sky.zip";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        TargetedMSChromatogramOptimizationTest init = (TargetedMSChromatogramOptimizationTest) getCurrentTest();
+        init.setupFolder(FolderType.Library);
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return "TargetedMS Chromatogram Optimization Test";
+    }
+
+    @Test
+    public void testUpload() throws Exception
+    {
+        goToProjectHome();
+        importData(SKY_FILE);
+        clickAndWait(Locator.linkContainingText("Panorama Dashboard"));
+        File downloadedClibFile = doAndWaitForDownload(() -> clickButton("Download", 0));
+
+        goToProjectHome();
+        String queryURL = "query-executeQuery.view?schemaName=targetedms&query.queryName=TransitionOptimization";
```

Comment:
Is there a reason this is navigating by direct URL? I was under the impression that we shouldn't be hard-coding URLs in tests except in very special circumstances

Reply:
Yes, i was navigating directly because there is was a case mismatch. Corrected the XML file.